tests/integration/test_filters.py

```diff
@@ -58,6 +58,55 @@ def test_filters_are_applied(
         events_consumer.get_event()
 
 
+@pytest.mark.parametrize(
+    "is_processing_relay", (False, True), ids=["external relay", "processing relay"]
+)
+@pytest.mark.parametrize(
+    "event, must_filter",
+    (
+        ({"logentry": {"message": "filter:"}}, False),
+        ({"logentry": {"message": "filter:yes"}}, True),
+        ({"logentry": {"message": "filter:%s", "params": ["no"]}}, False),
+        ({"logentry": {"message": "filter:%s", "params": ["yes"]}}, True),
```

Comment:
I have not added tests with `formatted` because I don't think they are valuable at this point. I'm open to adding them and it's not much additional work, so let me know if you think they provide enough value.